app/views/donation_mailer/_donation_payment_table.html.erb

```diff
@@ -84,23 +84,23 @@
 		<% @contact = JSON.parse(donation.dedication).dig('contact') %>
 		<tr>
 			<td><strong>Acknowledgement Email</strong></td>
-			<% if @contact['email'] %>
+			<% if @contact && @contact['email'] %>
```

Comment:
Does this work for all of these? It's a bit cleaner.

```suggestion
			<% if @contact&.dig('email') %>
```

Reply:
yep, looks like that works, thanks!